### PR TITLE
{tools}[system/system] vsc-mympirun v5.2.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3623.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/22971dcfda234211e80e33b8e826cb36 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3120.skitty.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/f62225bc0945ddea5e7431a027494aa9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3403.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/81febafa70ef38f393f6e616823507a4 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node101.hydra.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/851d724ef9e01c5bfc8083b4e6e65ace for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node371.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/58cdb069fe09c10900043159dc769c2e for a full test report.

File: easybuild/easyconfigs/v/vsc-mympirun/vsc-mympirun-5.2.2.eb [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+toolchain = SYSTEM
+
+allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
+
+use_pip = False
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('setuptools', '41.6.0', {
```
Comment (lexming) [inline easybuild/easyconfigs/v/vsc-mympirun/vsc-mympirun-5.2.2.eb:20]: Please add the previous comment about the restriction on `setuptools`, if `vsc-install` v0.17 still needs `setuptools` < 42.0.

**Review by lexming — CHANGES_REQUESTED** [review]

**Comment by verdurin** [comment]
@boegel any update here?

**Review by lexming — APPROVED** [review]
LGTM

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
nuc.lan - Linux Fedora 32, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.8.6
See https://gist.github.com/317d50a286ecd8dd87ffbd281e6b88d8 for a full test report.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Comment by lexming** [comment]
OMG I blinked and @verdurin already merged it :smile:

**Pull request merged**